packages/amplify_core/lib/src/types/api/graphql/graphql_response.dart

```diff
@@ -40,6 +40,6 @@ class GraphQLResponse<T> {
     );
   }
 
-  /// Whether errors are present in the response.
+  /// Returns true when errors are present and not empty.
```

Comment:
question: Was this change intended? maybe conflict error?

Reply:
Yes, please change back. Methods return, fields/getters hold information.